functest/main_test_wrapper.go

```diff
@@ -454,7 +485,7 @@ func teardown() {
 	fmt.Println("[ teardown ] directory for contracts cache was successfully deleted")
 }
 
-func testMainWrapper(m *testing.M) int {
+func TestsMainWrapper(m *testing.M) int {
```

Comment:
why you make it public? it also leads this test should have been run by `go test`

Reply:
the reason is to make this method to be called from other directories with test classes, not only from 'functest'. This way other tests could be runned in launchnet as well. QA already have other tests in separate branch

Reply:
@pavel-moskovkin  we need to extract the method body to another function if it needs, and use this function here. Just revert these changes, please